core/unix/signal.c

```diff
@@ -3158,6 +3162,14 @@ copy_frame_to_stack(dcontext_t *dcontext, int sig, sigframe_rt_t *frame, byte *s
         ASSERT_NOT_REACHED();
     }
 
+    kernel_sigset_t *pre_syscall_app_sigprocmask = NULL;
```

Comment:
The variable name seems odd for the common case of just delivering a signal: there's no syscall involved there.  Something like "mask_to_restore, or continuation_mask, or something seems clearer.

Reply:
Done.